File: LoadingBar.java
public class LoadingBar {
  private int total_perc; // loads the number that should be equal to 100%
  private int curr_status; // loads a current number. assuming it's under total_perc
  
  public LoadingBar(int total){
    this.total_perc = total;
    this.curr_status = 0;
  }

  public void update_curr_status(int new_status){
    this.curr_status = new_status;
  }

  public float get_perc(){
    return ((float)curr_status/(float)total_perc)*100;
  }

  public void show_loading_bar(){
    System.out.print("[");
    for(int j = 1; j <= 3; j++){
      if (j < (get_perc() / 10))
        System.out.print("=");
      else
        System.out.print(" ");
    }
    String perc = new String(get_perc().toString()+"%");
    System.out.print(perc);
    for(int j = 3+perc.length(); j <= 10; j++){
      if(j < (get_perc() / 10))
        System.out.print("=");
      else
        System.out.print(" ");
    }
    System.out.print("]\r");
  }
}

File: loadingBar.py
class LoadingBar:
    def __init__(self, total):
        self._total_perc = total
        self._curr_status = 0
    
    def update_curr_status(self, new_status):
        self._curr_status = new_status

    def get_perc(self):
        return (float(self._curr_status)/float(self._total_perc)) * 100

    def show_loading_bar(self):
        print("[", end="")
        for j in range(1,4):
            print("=" if j < (self.get_perc() / 10) else " ", end="")
        perc = str(get_perc)+"%"
        print(perc, end="")
        for j in range(3+len(perc), 11):
            print("=" if j < (self.get_perc() / 10) else " ", end="")
        print("]",end="\r")
